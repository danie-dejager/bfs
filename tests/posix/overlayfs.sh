test "$UNAME" = "Linux" || skip

clean_scratch
"$XTOUCH" -p scratch/{lower/{foo,bar,baz},upper/{bar,baz/qux}}

mkdir -p scratch/{work,merged}
bfs_sudo mount -t overlay overlay -olowerdir=scratch/lower,upperdir=scratch/upper,workdir=scratch/work scratch/merged || skip
defer bfs_sudo rm -rf scratch/work
defer bfs_sudo umount scratch/merged

bfs_diff scratch/merged
