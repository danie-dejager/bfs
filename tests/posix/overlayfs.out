scratch/merged
scratch/merged/bar
scratch/merged/baz
scratch/merged/baz/qux
scratch/merged/foo
